main/src/main/java/com/airbnb/reair/incremental/primitives/CopyUnpartitionedTableTask.java

```diff
@@ -102,6 +103,19 @@ public RunInfo runTask() throws HiveMetastoreException, DistCpException, IOExcep
 
     if (existingTable != null) {
       LOG.debug("Table " + spec + " exists on destination");
+
+      if (!conf.getBoolean(ConfigurationKeys.BATCH_JOB_OVERWRITE_NEWER, true)) {
+        Table freshDestTable = existingTable;
+        if (ReplicationUtils.isSrcOlder(freshSrcTable, freshDestTable)) {
+          LOG.warn("Source table " + spec + " ("
```

Comment:
Great, this log line will be more useful for debugging. Could you do the same for the two other places below?